plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -842,6 +843,13 @@ public MaterializedViewFreshness getMaterializedViewFreshness(ConnectorSession s
             return new MaterializedViewFreshness(false);
         }
 
+        return getMaterializedViewFreshness(session, refreshStateMap);
+    }
+
+    public MaterializedViewFreshness getMaterializedViewFreshness(ConnectorSession session, Map<String, Optional<TableToken>> refreshStateMap)
```

Comment:
why it's public since its unused? Should it be package private?

Reply:
Removed extra method.